tsconfig.json

```diff
@@ -42,6 +42,7 @@
     "noFallthroughCasesInSwitch": true,
     "noUncheckedIndexedAccess": true,
     "inlineSources": true,
+    "useDefineForClassFields": false,
```

Comment:
This option disable es2022 new feature https://github.com/tc39/proposal-class-fields when emiting